build.sh

```diff
@@ -50,20 +50,22 @@ usage() {
 Usage: $0 <options>
   Optional options:
      --be               build Backend
-     --fe               build Frontend
+     --fe               build Frontend and Spark Dpp application
```

Comment:
The default build FE should without Spark Dpp application?

Reply:
Can not, FE depends on some class in SparkDpp. Maybe we can move them to fe-common later.

Reply:
OK